### PR TITLE
Fix diskusage crash on unsupported platforms + add freebsd

### PR DESCRIPTION
**Pull request by inve1**
When using flood on an unsupported platform, the app wasn't loading because of this exception:

```
GET /api/activity-stream?historySnapshot=fiveMin 200 1.189 ms - -
TypeError: diskUsage[process.platform] is not a function
    at DiskUsageService.updateDisks (/usr/home/inve1/flood/server/services/diskUsageService.js:99:39)
    at module.exports (/usr/home/inve1/flood/server/middleware/clientActivityStream.js:46:20)
    at Layer.handle [as handle_request] (/usr/home/inve1/flood/node_modules/express/lib/router/layer.js:95:5)
    at next (/usr/home/inve1/flood/node_modules/express/lib/router/route.js:137:13)
    at module.exports (/usr/home/inve1/flood/server/middleware/eventStream.js:26:3)
    at Layer.handle [as handle_request] (/usr/home/inve1/flood/node_modules/express/lib/router/layer.js:95:5)
    at next (/usr/home/inve1/flood/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/usr/home/inve1/flood/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/usr/home/inve1/flood/node_modules/express/lib/router/layer.js:95:5)
    at /usr/home/inve1/flood/node_modules/express/lib/router/index.js:281:22
```

## Description
I put in a check for supported platforms on that function. Since the freebsd df output is the same as linux I also added it as a supported platform with a small refactor

A note about the `maxBuffer` change: My df output is currently 31045 bytes long. In my case this is because of hundreds of rar2fs mounts to make rar downloads plex-friendly. Perhaps filtering filesystems like these before parsing them in node could be an option since the disk usage from these is useless. Don't want to handle this in a bugfix pr though.

## Motivation and Context
Flood stopped working on freebsd after I pulled master today 

## How Has This Been Tested?
Ran it on freebsd

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: server/services/diskUsageService.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ const execFile = util.promisify(require('child_process').execFile);
 const config = require('../../config');
 const diskUsageServiceEvents = require('../constants/diskUsageServiceEvents');
 
-const PLATFORMS_SUPPORTED = ['darwin', 'linux'];
+const PLATFORMS_SUPPORTED = ['darwin', 'linux', 'freebsd'];
```
Comment (noraj) [inline server/services/diskUsageService.js]:
```diff
-freebsd
+bsd
```

As it is Linux and not a distro name.

Comment (inve1) [reply server/services/diskUsageService.js]: Sure but process.platform returns the string ‘freebsd’. Also I wouldn’t assume the output of df is consistent across all BSDs. technically Darwin is a BSD and the output is different :P

Comment (noraj) [reply server/services/diskUsageService.js]: @inve1 You are totally right, my bad.

**Comment by noraj** [comment]
LGTM : Looks Good To Me (I had to search lol)

**Pull request merged**